test/metabase/api/automagic_dashboards_test.clj

```diff
@@ -156,3 +161,15 @@
                (->> [:= [:field-id (data/id :venues :price)] 15]
                     (#'magic/encode-base64-json))
                segment-id])))
+
+
+;;; ------------------- Transforms -------------------
+
+(expect
+  (with-rasta
```

Comment:
I don't think this is really an adequate test (just making sure the function returns something and doesn't throw an Exception). We should have some tests that actually check that we get some output that looks like what we'd expect given some input. You should be able to mock things with `with-redefs` as needed